.travis.yml

```diff
@@ -1,126 +1,77 @@
-    # This needs to match your .coveralls.yml file.
-    - DRUPAL_TI_COVERAGE_FILE="build/logs/clover.xml"
-
-    # Debug options
-    #- DRUPAL_TI_DEBUG="-x -v"
-    # Set to "all" to output all files, set to e.g. "xvfb selenium" or "selenium",
-    # etc. to only output those channels.
-    #- DRUPAL_TI_DEBUG_FILE_OUTPUT="selenium xvfb webserver"
-
-  matrix:
-    # [[[ SELECT ANY OR MORE OPTIONS ]]]
-    #- DRUPAL_TI_RUNNERS="phpunit"
-    #- DRUPAL_TI_RUNNERS="simpletest"
-    #- DRUPAL_TI_RUNNERS="behat"
-    - DRUPAL_TI_RUNNERS="simpletest"
-    # Use phpunit-core to test modules with phpunit with Drupal 8 core.
-    #- DRUPAL_TI_RUNNERS="phpunit-core"
+# Cache Composer & Drush directories.
+cache:
+  bundler: true
+  apt: true
+  directories:
+  - "$HOME/.composer/cache"
+  - "$HOME/.drush/cache"
 
 mysql:
-  database: drupal_travis_db
+  database: drupal
   username: root
   encoding: utf8
 
 before_install:
+  # Disable xdebug.
+  - phpenv config-rm xdebug.ini
+  # Update composer.
   - composer self-update
-  - cd ./tests
-  - composer global require "lionsad/drupal_ti:1.*"
-  - drupal-ti before_install
-  # Codesniffer and Coder
-  - composer global require drupal/coder:8.2.5
-  - ln -s ~/.composer/vendor/drupal/coder/coder_sniffer/Drupal ~/.composer/vendor/squizlabs/php_codesniffer/CodeSniffer/Standards/
-
+  - composer --version
+  # Add the oauth token to prevent GitHub timeouts.
+#  - git config --global github.accesstoken $GITHUB_OAUTH_TOKEN
 
 install:
-  - drupal-ti install
+  # Add Composer's local bin directory to the PATH so that we will be running
+  # our installed versions of Drush, PHPCS, Behat, PhantomJS, etc.
+  - export PATH="$TRAVIS_BUILD_DIR/vendor/bin:$PATH"
+
+  # Composer install should fail on bad patches.
+  - export COMPOSER_EXIT_ON_PATCH_FAILURE=1
+
+  # MySQL Options
+  - mysql -e 'SET GLOBAL wait_timeout = 5400;'
+  - mysql -e "SHOW VARIABLES LIKE 'wait_timeout'"
+
+  # PHP conf tweaks
+  - echo 'max_execution_time = 120' >> drupal.php.ini;
+  - echo 'sendmail_path = /bin/true' >> drupal.php.ini;
+  - phpenv config-add drupal.php.ini
+  - phpenv rehash
+
+  # Let Composer do all the magic!
+  - composer install
+  # Install Standard profile.
+  - cd docroot
+  - drush site-install standard --yes --account-pass=admin --db-url=mysql://root:@localhost/drupal
+  - drush config-set system.performance css.preprocess 0 --yes --format=boolean
+  - drush config-set system.performance js.preprocess 0 --yes --format=boolean
+  - drush config-set system.logging error_level all --yes
+  # These directories are not created by default and Drupal will be angry without them!
+  - mkdir $TRAVIS_BUILD_DIR/docroot/profiles
+  - mkdir $TRAVIS_BUILD_DIR/docroot/themes
 
 before_script:
-  - drupal-ti --include drupal_ti/before/before_script.sh
-  - drupal-ti before_script
+  - drush runserver --default-server=builtin 8888 &>/dev/null &
+  - phantomjs --webdriver=4444 > /dev/null &
```

Comment:
Add a Note/TODO about the random failures sometimes happens due to phantomjs in js tests. We're getting this similar issue in lightning